src/pages/login/Login.jsx

```diff
@@ -115,7 +115,7 @@ const Login = () => {
                       name="password"
                       placeholder="Type here..."
                       {...register("password", {
-                        required: "Password is required",
+                        required: "Invalid Password",
```

Comment:
This should not be invalid password

Required field cannot be empty!  Change it

Reply:
Changed the error message name.